src/main/java/ch/njol/skript/effects/EffLeash.java

```diff
@@ -34,8 +34,11 @@
 import ch.njol.util.Kleenean;
 
 @Name("Leash entities")
-@Description("Leash living entities to other entities.")
-@Examples("leash the player to the target entity")
+@Description("Leash living entities to other entities. When trying to leash an Ender Dragon, Wither, Player, or a Bat, this effect will not work.")
+@Examples({
+			"on right click:",
+				"\tleash event-entity to player",
+				"\tsend \"&aYou leashed &2%event-entity%!\" to player"})
```

Comment:
Quick correction to indentation, we use 1 indentation
```suggestion
@Examples({
	"on right click:",
		"\tleash event-entity to player",
		"\tsend \"&aYou leashed &2%event-entity%!\" to player"
})
```